src/shared/SidebarTemplate.tsx

```diff
@@ -0,0 +1,56 @@
+import { Flex, Box } from '@chakra-ui/react';
+import { useState, useEffect } from 'react';
+import { Helmet } from 'react-helmet';
+import { useLocation, useNavigate } from 'react-router';
+
+import { Header, Feedback } from '../app';
+import { getCurrentTerm } from '../app/shared/utils/terms';
+import { ContentSidebar } from '../app/sidebar';
+
+import { Term } from './fetchers';
+import { useSessionStorage } from './hooks/storage/useSessionStorage';
+
+export interface SidebarTemplateProps {
+  title: string;
+  children: JSX.Element;
+  term: Term;
+}
+
+export function SidebarTemplate({ children, term, title }: SidebarTemplateProps): JSX.Element {
```

Comment:
I'd be more favoured to use `PropsWithChildren` so we don't have to pass in the `child` as a prop but rather a child node.
https://www.newline.co/@bespoyasov/how-to-define-props-with-children-in-react-typescript-app--56bd18be